metaflow/plugins/debug_logger.py

```diff
@@ -1,17 +1,28 @@
 import sys
 
-from metaflow.sidecar_messages import Message
+from metaflow.event_logger import BaseEventLogger
+from metaflow.sidecar import Message, MessageTypes
 
 
-class DebugEventLogger(object):
+class DebugEventLogger(BaseEventLogger):
     TYPE = "debugLogger"
 
-    def log(self, msg):
-        sys.stderr.write("event_logger: " + str(msg) + "\n")
+    @classmethod
+    def get_worker(cls):
+        return DebugEventLoggerSidecar
+
+
+class DebugEventLoggerSidecar(object):
+    def __init__(self):
+        pass
 
     def process_message(self, msg):
         # type: (Message) -> None
-        self.log(msg.payload)
+        if msg.msg_type == MessageTypes.SHUTDOWN:
+            print("Debug[shutdown]: got shutdown!", file=sys.stderr)
+            self._shutdown()
+        elif msg.msg_type == MessageTypes.BEST_EFFORT:
+            print("Debug[event]: %s" % str(msg.payload), file=sys.stderr)
```

Comment:
also print the `MUST_SEND_EVENT` message?

Reply:
fixed.